x-pack/plugins/security/server/authorization/privileges/feature_privilege_builder/alerting.ts

```diff
@@ -16,7 +16,7 @@ enum AlertingEntity {
 }
 
 const readOperations: Record<AlertingEntity, string[]> = {
-  rule: ['get', 'getRuleState', 'getAlertSummary', 'find'],
+  rule: ['get', 'getRuleState', 'getAlertSummary', 'getExecutionLog', 'find'],
```

Comment:
:+1:

Don't forget to update `x-pack/plugins/alerting/README.md` to mention this new API!

Reply:
Good catch! Updated here: https://github.com/elastic/kibana/pull/127339/commits/1fad8678bf566dbc00b78d1e043c3d95e331f5b0